lib/connectors/kv-memory.js

```diff
@@ -120,6 +124,27 @@ function(modelName, key, ttl, options, callback) {
   process.nextTick(callback);
 };
 
+KeyValueMemoryConnector.prototype.ttl =
+function(modelName, key, options, callback) {
+  this._removeIfExpired(modelName, key);
+
+  var store = this._getStoreForModel(modelName);
+
+  // key is unknown
+  if (!(key in store)) {
+    return process.nextTick(function() {
+      callback(new Error('Cannot get TTL for unknown key: ' + key));
```

Comment:
Please set `err.statusCode = 404` so that the REST API returns `404 Not Found` when this error happens.